src/test/java/Pages/homePage.java

```diff
@@ -0,0 +1,110 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class homePage extends basePage {
+    WebDriver driver;
+    static WebDriverWait wait;
```

Comment:
**Please, remove code redundancies**

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )